cli/installer/install-azd.ps1

```diff
@@ -279,16 +266,16 @@ if (isLinuxOrMac) {
     }
 
     $params = @(
-        '--base-url', "`"$BaseUrl`"", 
-        '--version', "`"$Version`""
+        '--base-url', "'$BaseUrl'", 
+        '--version', "'$Version'"
```

Comment:
I've seen some shells - e.g., older cmd.exe - fail on single quotes. Have you tested this on Windows if cmd.exe is used, or is that not a case since this is running directly in PS?